Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -235,6 +235,7 @@ @interface ConversationViewController () <AttachmentApprovalViewControllerDelega
 @property (nonatomic) ContactShareViewHelper *contactShareViewHelper;
 @property (nonatomic) NSTimer *reloadTimer;
 @property (nonatomic, nullable) NSDate *lastReloadDate;
+@property (nonatomic, nullable) NSDate *collapseCutoffDate;
```

Comment:
Don't collapse footers of "very new" messages, e.g. messages:

* Messages created since we entered the view _and_ since we last did `resetContentAndLayout`, e.g. we re-entered the view or the "refresh timer" fired.